### PR TITLE
Sugarcoat resources

### PR DESCRIPTION
**Pull request by ShivanKaul**

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,8 +1,26 @@
 [
-{
-    "name": "brave-fix",
-    "aliases": ["bf"],
```
Comment (ShivanKaul) [inline metadata.json:4]: Do we need this alias?

Comment (antonok-edm) [reply metadata.json:4]: @ryanbr

**Review by antonok-edm — APPROVED** [review]
Looks good to me but let's confirm with @ryanbr about the `bf` alias before merging

**Comment by ryanbr** [comment]
Yeah we still need "bf" @ShivanKaul

**Comment by ShivanKaul** [comment]
@ryanbr gotcha, added that back in. It would be nice if we could generate the alias automatically; I don't want to retrofit an alias for every sugarcoat resource just because of brave-fix though, so leaving it as a manual change, added a note to `generateMetadataJsonFromScriptResources.js` that a manual fixup is needed after running it.

**Review by antonok-edm — APPROVED** [review]

**Review by antonok-edm — APPROVED** [review]

**Pull request merged**